src/appEntry.css

```diff
@@ -479,31 +461,54 @@ textarea:focus {
     opacity: 0.6;
 }
 
+/* ADMIN PANEL ------------------------------------------------------------ */
```

Comment:
These comments are a signal that it might be a good opportunity to move styles related to the admin panel into their own stylesheet.